query.go

```diff
@@ -796,3 +824,136 @@ func scanRow(cols []string, rows *sql.Rows) (Result, error) {
 
 	return result, nil
 }
+func (q *Query) UpdateTransaction(resources []Txn) (err error) {
+
+	txn, err := database.SQLDB().Begin()
+	if err != nil{
+		return err
+	}
+	var index int
+	index = 1
+	var finalStatment string
+	finalStatment = "with "
+	var finalParams []interface{}
+
+
+	for i, resource := range resources {
+
+		var params []interface{}
+		var sql string
+
+		operation := resource.Operation
+		//Generate sql comman
+		if operation == "update"{
```

Comment:
Did we write this code or it is from upstream?

Reply:
we write it

Reply:
Then we need to rewrite it in a better way. I will help